### PR TITLE
[Keymap] Workman layout for Atreus keyboard

### PR DESCRIPTION
**Pull request by kensanata**
This adds an implementation of the Workman layout for Atreus keyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by kensanata** [comment]
The only thing I'm unsure about is that for my own use, I've had to swap CTL and ALT in the keymap configuration in order to get the documented behaviour. My assumption is that this is bug elsewhere and thus I'm not including 79ae59c155c76394b09fb4286951a6ed5b71d82d in this pull request. As I also have the older version of the keyboard and need to compile with PCBDOWN=yes, perhaps the issue is also hidden in there somewhere. I have no idea.
Either way, I think keymap would make a good starting point.

File: keyboards/atreus/keymaps/workman/config.h [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#define TAPPING_TOGGLE 1
+#define ONESHOT_TAP_TOGGLE 1
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline keyboards/atreus/keymaps/workman/config.h]: This setting is the default, now, and does not need to be included anymore.

```suggestion
```

**Comment by kensanata** [comment]
Sure!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**